src/psyclone/transformations.py

```diff
@@ -2867,8 +2832,10 @@ def _validate(self, sched):
                          (subclass of) :py:class:`psyclone.psyGen.Schedule`.
         '''
         from psyclone.psyGen import Directive, \
```

Comment:
pylint complains that `ACCKernelsDirective` and `ACCParallelDirective` are not used.

Reply:
Done.